unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Utils/UiElementUtils.cs

```diff
@@ -10,7 +10,7 @@ public static void SetElementDefaultStyle(IStyle elementStyle)
         elementStyle.bottom = 0;
         elementStyle.width = new StyleLength(StyleKeyword.Auto);
         elementStyle.height = new StyleLength(StyleKeyword.Auto);
-        elementStyle.position = new StyleEnum<Position>(Position.Relative);
+        elementStyle.position = new StyleEnum<Position>(Position.Absolute);
```

Comment:
mm its weird.
We are sending the position: relative by default on the sdk level 🤔 
https://github.com/decentraland/js-sdk-toolchain/blob/main/packages/%40dcl/react-ecs/src/components/uiTransform/index.ts#L37

Reply:
that change right now only affects the UiText, and it only affects the inner visual element of the UiText.
Some UiComponents we create are actually composed of 2 visual elements: a root element that acts as a kind of container and an inner one. UiText has a container and a label as the child.

We were already setting this as Absolute before I implemented the auto-size support.